src/styles/header.css

```
@@ -0,0 +1,46 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.header{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom:1.875rem;
+    padding-bottom: 0.938rem;
+    border-bottom: 1px solid var(--border-color);
+    margin: 1.25rem;
+}
+
+h1{
+    color: var(--primary-color);
+    margin: 0;
+}
+
+.btn-controller{
+    display: flex;
+    align-items: center;
+    gap:1.5rem;
+}
+
+#toggle-btn , #expense-btn{
+    background: var(--primary-color);
+    border: none;
+    color: #ffffff;
+    cursor: pointer;
+    padding: 8px 15px;
```

Can we use `rem` instead of px
```suggestion
    padding: 0.5rem 0.95rem;
```

---

add https://github.com/dhruvdhaduk-simform/fair-share/pull/5/commits/c6c1fbb58e9420bf4bec7ce1e6dc9f296ca2c9dc